handler/commentHandler_test.go

```diff
@@ -515,3 +515,44 @@ func Test_isDcoLabel(t *testing.T) {
 		})
 	}
 }
+
+func Test_Parsing_Reviewers(t *testing.T) {
+
+	var reviewersOptions = []struct {
+		title        string
+		body         string
+		expectedType string
+		expectedVal  string
+	}{
+		{
+			title:        "Right set reviewer",
+			body:         "set reviewer: johndoe",
```

Comment:
Just nit picking. `johndoe` can be used in some negative contexts so we could just put something else like `john` on its own?